m4_sessions/1810-inning/ethics_of_ai.md

```diff
@@ -1,31 +1,38 @@
 ## Group Members:
 
-
+Connor Osburn
+Chris Powell
+Tom King
+Brandon Fiebiger
+Benjamin Paige
 
 ## Name of Session: 
 
+Ethics of AI
+
 ## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
 
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
 
+     ## general undersanding of what it is     
+     ## Know what AI is and what the ethical probloms are. 
+          - now
+          - future
+     ## high level view, where it came from and what the future looks like.
 
-
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
 ## Session Outline 
 
 (Please include who is facilitating and how much time you are going to spend in each section):
 
-## Recommended Trajectory 
-
-    ## Introductons 
-    ## Intro to new material
-    ## Small Group Activity
-    ## Closing Discussion
-
+Personal Intros 10min
```

Comment:
You could have them write on the board or you could write the ideas of AI on the board as a different way to open up